kuzzleio/kuzzle-admin-console
# Migrate e2e to cypress

## What does this PR do ?

We finally decided to use Cypress.io for e2e testing. This PR sets-up the stack to run the tests in Cypress.

### How should this be manually tested?

1. `npm run e2e-local`, the local dashboard will appear
2. pick a spec file and run it locally

or

1. `npm run e2e`
2. go to https://dashboard.cypress.io/#/projects/im2yeo/runs (use your GitHub handle to login)
3. take a look at the results
